src/bionics_ui.cpp

```diff
@@ -36,6 +37,94 @@ enum bionic_menu_mode {
     EXAMINING,
     REASSIGNING
 };
+
+enum bionic_sort_mode {
+    POWER,
+    NAME,
+};
+bionic_sort_mode _bionic_sort_mode = bionic_sort_mode::POWER;
```

Comment:
This probably shouldn't be a global variable.  We have some other global UI state (e.g. the last state of the advanced Inventory screen).  I'm not sure where that get stored, but this could perhaps go somewhere similar?

Reply:
This is in the anonymous namespace so it only exists in this translation unit

Reply:
I've managed to track this down and moved the variable to `uistate`. Thanks for pointing me in the right direction